nservicebus/azure-service-bus/batching.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Azure Service Bus Transport Batching
+summary: Batching with Azure Service Bus and how it works
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports
+reviewed: 2016-04-26
+---
+
+## Batching store access
+
+To increase entities throughput, Azure Service Bus batches messages prior to writing to its internal store. Write message operations are delayed by 20ms and that helps to increase throughput. Read operations are not affected by this setting. By default batching store access is enabled. To change the default, set `EnableBatching` on [queues](/nservicebus/azure-service-bus/configuration/configuration.md#Queues), [topics](/nservicebus/azure-service-bus/configuration/configuration.md#Topics), and [subscriptions](/nservicebus/azure-service-bus/configuration/configuration.md#Subscriptions).
+
+For low throughput scenarios with required low latency, batching store operations should be disabled.
+
```

Comment:
Not sure I like the 1st paragraph. It reads as if we encourage to turn batching off, where we do the opposite. Will slightly rephrase.